beast/tests/helpers.py

```diff
@@ -38,6 +38,9 @@ def compare_tables(table_cache, table_new):
     table_cache : astropy table
     table_new : astropy table
         data for comparision.
+    rtol : float (default=1e-7)
+        relative tolerance for np.testing.assert_allclose (default of 1e-7
+        matches the default in assert_allclose)
     """
     if not len(table_new) == len(table_cache):
         raise AssertionError()
```

Comment:
Shouldn't there be an error message with this error?  Usually done to provide a bit more info to the user.

Reply:
I know this is not the code you wrote - I'm sure I did.  But could you add one?  Or is that not possible with an AssertionError?

Reply:
Looks like it's possible, so I'll go add a quick few words :)